packages/functional-tests/tests/syncV3/fxDesktopV3ForceAuth.spec.ts

```diff
@@ -93,8 +93,8 @@ test.describe('severity-1 #smoke', () => {
     }) => {
       const config = await configPage.getConfig();
       test.skip(
-        config.showReactApp.signUpRoutes === true,
-        'force_auth is no longer supported for signup with react'
+        config.showReactApp.resetPasswordRoutes === true,
+        'Scheduled for removal as part of React conversion (see FXA-8267).'
```

Comment:
Are you sure about changing this flag?

```suggestion
        config.showReactApp.signUpRoutes === true,
        'force_auth is no longer supported for signup with react'
```

Reply:
Ah, I see what you mean. Yeah it should be the same flag. But I am going to create a Jira ticket to remove the force_auth tests so that I can link it in the skip comments here.

Reply:
FXA-9410 added to the comments.